File: app/src/main/java/nl/rsdt/japp/jotial/maps/wrapper/IUiSettings.java
package nl.rsdt.japp.jotial.maps.wrapper;

/**
 * Created by mattijn on 09/08/17.
 */

public interface IUiSettings {

    void setAllGesturesEnabled(boolean allesturesEnabled);

    void setCompassEnabled(boolean compassEnabled);

    void setZoomControlsEnabled(boolean zoomControlsEnabled);

    void setIndoorLevelPickerEnabled(boolean indoorLevelPickerEnabled);

    void setMapToolbarEnabled(boolean mapToolbarEnabled);
}

File: app/src/main/java/nl/rsdt/japp/jotial/maps/wrapper/IUiSettings.kt
package nl.rsdt.japp.jotial.maps.wrapper

/**
 * Created by mattijn on 09/08/17.
 */

interface IUiSettings {

    fun setAllGesturesEnabled(allesturesEnabled: Boolean)

    fun setCompassEnabled(compassEnabled: Boolean)

    fun setZoomControlsEnabled(zoomControlsEnabled: Boolean)

    fun setIndoorLevelPickerEnabled(indoorLevelPickerEnabled: Boolean)

    fun setMapToolbarEnabled(mapToolbarEnabled: Boolean)
}
